src/common/options/rgw.yaml.in

```diff
@@ -3805,3 +3805,34 @@ options:
   flags:
   - startup
   with_legacy: true
+- name: rgw_topic_persistency_time_to_live
+  type: uint
+  level: advanced
+  desc: The rgw retention of persistent topics by time (seconds)
+  default: 4294967295
```

Comment:
Why is such the highest limit set on default values for both options? No other rgw options have such tremendous values for expiration time or retries

Reply:
we should probably have default of zero - which should indicate "no expiration".
so, we don't change the current behavior

Reply:
we should add the "zero" case to the description